content/docs/user-guide/assets/asset-processor/asset-cache-server.md

```diff
@@ -0,0 +1,105 @@
+---
+linkTitle: Asset Cache Server Mode
+title: Asset Cache Server
+description: Enable a feature to reduce asset builds for a team.
+weight: 200
+toc: true
+---
+
+The **Asset Server Cache** mode is a feature allows an **Asset Processor** to cache product asset files to a remote folder so that other team members can skip long asset processing times.
```

Comment:
```suggestion
The **Asset Server Cache** feature allows **Asset Processor** to cache product asset files to a remote folder so that other team members can skip long asset processing times.
```

Reply:
The ""Asset Server Cache"" is a mode that a user puts the AP into so that it can cache stuff. What is the reasoning to drop the "mode" word?

Reply:
That's fine, the original wording was confusing.  Perhaps "The asset server cache mode allows ..." would match how the next sentence starts